src/main/kotlin/io/openfuture/snapshot/dto/SnapshotRequest.kt

```diff
@@ -0,0 +1,8 @@
+package io.openfuture.snapshot.dto
+
+data class SnapshotRequest(
```

Comment:
you should not wrap these parameters into the class. Would be enough to pass them separated by comma